runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDoViaKeyedWorkItems.java

```diff
@@ -87,6 +89,12 @@
     public String getUrn() {
       return SplittableParDo.SPLITTABLE_GBKIKWI_URN;
     }
+
+    @Nullable
+    @Override
+    public RunnerApi.FunctionSpec getSpec() {
```

Comment:
<!--new_thread; commit:a9a58870c39402cb77114fb054b85962a63a31a0; resolved:0-->
Should this be NotSerializable?